### ISSUE TITLE
Accessibility: governance/papers

### ISSUE THREAD

**Issue by Bhaybhiisparks** [jupyterhub/outreachy#120]
### What page is this for?

https://jupyter.org/governance/papers.html

### WAVE accessibility report

https://github.com/jupyterhub/outreachy/issues/38#issuecomment-1288208023

<img width="960" alt="papershtm" src="https://user-images.githubusercontent.com/86384288/197420915-db8a9322-1134-4bbb-b72c-ee01990de4fc.png">

### The HTML element to be changed

```html
1. Missing Document Language:
<html style="margin-left: 380px;"> 

2.Multiple Form Labels:
--  <input type="checkbox" class="sidebar-toggle" name="__page-toc" id="__page-toc" aria-label="Toggle in-page Table of Contents">
<label class="overlay overlay-pagetoc" for="__page-toc">
<div class="visually-hidden">
Toggle in-page Table of Contents
</div>

-- <input type="checkbox" class="sidebar-toggle" name="__navigation" id="__navigation" aria-label="Toggle navigation sidebar">
<label class="overlay overlay-navbar" for="__navigation">
<div class="visually-hidden">
Toggle navigation sidebar
</label>
</div>

3. Linked Image with alternative Text:
<img src="https://licensebuttons.net/p/zero/1.0/88x31.png" style="border-style: none;" alt="CC0">

4. Form label is present but a non form element is encapsulated within
-- <label class="overlay overlay-navbar" for="__navigation">
<div class="visually-hidden">
Toggle navigation sidebar
</div>
</label> 

-- <label class="overlay overlay-pagetoc" for="__page-toc">
<div class="visually-hidden">
Toggle in-page Table of Contents
</div>
</label>
```

### Your proposed HTML (or CSS), after change

```html
1. Define the language attribute
<html style="margin-left: 380px;" lang="en"> 

2. The labels do not appear to be in association with more than one form element and form control, but wave might be reading the div encapsulated within the label itself as a form control associated with the label. A possible fix would be to encapsulate the label and the input elements inside the div rather than the other way round. 

--  <div class="visually-hidden">
 <input type="checkbox" class="sidebar-toggle" name="__page-toc" id="__page-toc" aria-label="Toggle in-page Table of Contents">
<label class="overlay overlay-pagetoc" for="__page-toc">
</div>

-- <div class="visually-hidden">
<input type="checkbox" class="sidebar-toggle" name="__navigation" id="__navigation" aria-label="Toggle navigation sidebar">
<label class="overlay overlay-navbar" for="__navigation">
Toggle navigation sidebar
</label>
</div>

3. Give the alternate (alt) attribute a value of null ("")
<img src="https://licensebuttons.net/p/zero/1.0/88x31.png" style="border-style: none;" alt="">

4. Solution 2 solves this issue.
<label class="overlay overlay-navbar" for="__navigation">
<div class="visually-hidden">
Toggle navigation sidebar
</div>
</label>
```

**Comment by minrk** [comment]
Thanks for the suggestions!

**Comment by sgibson91** [comment]
Thank you for your contribution! Since the contribution period is now over, we will close this issue.
